cmd/sequencer/getBlockheight.go

```diff
@@ -0,0 +1,35 @@
+package sequencer
+
+import (
+	"fmt"
+
+	"github.com/astria/astria-cli-go/cmd"
+	"github.com/astria/astria-cli-go/internal/sequencer"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// getBlockHeightCmd represents the get-blockheight command
+var getBlockHeightCmd = &cobra.Command{
+	Use:    "get-blockheight",
+	Short:  "Retrieves the latest blockheight from the sequencer.",
+	PreRun: cmd.SetLogLevel,
+	Run:    runGetBlockHeight,
+}
+
+func init() {
+	sequencerCmd.AddCommand(getBlockHeightCmd)
+	getBlockHeightCmd.Flags().String("url", DefaultSequencerURL, "The URL of the sequencer to retrieve the balance from.")
+}
+
+func runGetBlockHeight(cmd *cobra.Command, args []string) {
```

Comment:
Nit: let's be consistent with the naming of blockheight. Here it's cased as two words but I think it's one word, like in the command and filename.